packages/babel-plugin/src/utils/ast-builders.tsx

```diff
@@ -79,6 +79,9 @@ const styledStyleProp = (
   return t.objectExpression(props.concat(buildCssVariablesProp(variables, transform)));
 };
 
+const buildComponentTag = ({ name, type }: Tag) =>
+  type === 'InBuiltComponent' ? `"${name}"` : name;
```

Comment:
are we imagining using this data later to do some transformations? there would be an opportunity to simplify our implementation atm if we just make `styled.tsx` pass in either `"name"` or `name` depending on the found code

Reply:
Yeah..being explicit is better than implicit. I thought this is an ast builder file and building logic should reside here. `styled.tsx` should only give extracted data.